src/backend/app/Http/Controllers/SolarHomeSystemController.php

```diff
@@ -45,4 +45,10 @@ public function search(Request $request): ApiResource {
 
         return ApiResource::make($this->solarHomeSystemService->search($term, $paginate));
     }
+
+    public function show($id): ApiResource {
+        $solarHomeSystem = $this->solarHomeSystemService->getById($id);
+
+        return ApiResource::make($solarHomeSystem->load(['manufacturer', 'appliance', 'device.person', 'device.address.geo']));
```

Comment:
Is this really required here? I.e. doesn't all this information already get loaded in `solarHomeSystemService->getById($id);`? The `device.address.geo` isn't there, but can't we just add it to the service?

The background is: It would be nice if all the "logic" can be in the service. This would be consistent with most other "get by id" endpoints where they just call the corresponding service's `getById` method.

Reply:
Oh, rightly spotted. I will move it into the service